machine/v1beta1/types_vsphereprovider.go

```diff
@@ -70,6 +70,15 @@ type VSphereMachineProviderSpec struct {
 	// When using LinkedClone, if no snapshots exist for the source template, falls back to FullClone.
 	// +optional
 	CloneMode CloneMode `json:"cloneMode,omitempty"`
+	// dataDisks is a list of non OS disks to be created and attached to the VM.  The max number of disk allowed to be attached is
+	// currently 15.  This limitation is being applied to allow no more than 16 disks on the default scsi controller for the VM.
+	// The first disk on that SCSI controller will be the OS disk from the template.
+	// +openshift:enable:FeatureGate=VSphereMultiDisk
+	// +optional
+	// +listType=map
+	// +listMapKey=name
+	// +kubebuilder:validation:MaxItems=29
```

Comment:
The godoc says 15 disks maximum, so why is this limit 29?

Reply:
I'll update this godoc.  Upstream didn't describe all this so I'll clean it up to match.